setup.cfg

```diff
@@ -1,9 +1,143 @@
+[metadata]
+name = evalml
+version = attr: evalml.__version__
+author = Alteryx, Inc.
+license = BSD 3-clause
+license_files = 
+    LICENSE
+long_description = file: READEME.md
+url = https://github.com/alteryx/evalml/
+download_url = https://pypi.org/project/evalml/
+long_description_content_type = text/markdown
+author_email = open_source_support@alteryx.com
+keywords = data science machine learning optimization automl
+description = EvalML is an AutoML library that builds, optimizes, and evaluates machine learning pipelines using domain-specific objective functions.
+
+classifiers =
+    Development Status :: 3 - Alpha
```

Comment:
I think it's more appropriate to mark it at least [beta](https://pypi.org/classifiers/) (we don't have a 1.0 release so maybe not "stable" yet).